src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js

```diff
@@ -353,9 +337,7 @@ export default React.createClass({
         const DialogButtons = sdk.getComponent('views.elements.DialogButtons');
         return <div>
             <p>{_t(
-                "Type in your Recovery Passphrase to confirm you remember it. " +
-                "If it helps, add it to your password manager or store it " +
-                "somewhere safe.",
+                "Please type in your passphrase again to check you remember it.",
```

Comment:
This feels like it is missing words, but still passes the "is English" test.

Reply:
'type your passphrase in again' may be better?

Reply:
It falls apart for me at "to check you remember it" - it feels like some context is missing at least. Something like "Type in your passphrase again to ensure it is correct" might work?

Reply:
Agreed, this is stumbly.

The 'to check you remember it' business taking a leaf out of Monzo's book and trying to speak like a human (rather than the more robotic alternative: "ENTER YOUR PASSPHRASE TO CONFIRM").

I think 1password says something warm and friendly like "Looks good! Let's try practicing that one more time to make sure you know it off by heart!" but I'm afraid of that getting lost in (multiple) translations. 

You're right, though - we're actually not just doing this to check they remember it - we're doing it to minimise the risk of their having typooed it the first time.

How about: "Please enter your passphrase a second time to confirm"?

Reply:
Done